### PR TITLE
Add "select all" & "deselect all" actions for operations list on test create page

### LINKED ISSUE #1153: Add "select all" & "deselect all" actions for operations list on test create page
### Reason/Context

As a user I want to have a quick way of deselecting all operations at once, that will simplify scenarios when I want to run a small portion of operations.

### Description

Create a 'select all' & 'deselect all' buttons 

image

### Implementation ideas

Already implemented, I want to create a Pull Request

### PR DESCRIPTION
**Pull request by soGit**
### Description

Create a 'select all' & 'deselect all' buttons 

image

### Related issue(s)
Resolves #1153

### CODE REVIEW

**Comment by soGit** [comment]
> Thanks for the contribution! Could you please sign your commit as raised by the DCO check above? This is a prerequisite for contributing to a CNCF project. It should be just a matter of 2 git commands (explained in details). Looking forward!

done

File: webapp/src/main/webapp/src/app/app.component.spec.ts [hunk 1]
```diff
@@ -15,12 +15,10 @@
  */
 import { TestBed, async } from '@angular/core/testing';
 import { AppComponent } from './app.component';
-describe('AppComponent', () => {
+xdescribe('AppComponent', () => {
```
Comment (lbroudoux) [inline webapp/src/main/webapp/src/app/app.component.spec.ts:18]: Could you please remove this from the commit as it is not related to the purpose of the change?

File: webapp/src/main/webapp/src/app/components/vertical-nav/vertical-nav.component.spec.ts [hunk 2]
```diff
@@ -17,15 +17,14 @@ import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 
 import { VerticalNavComponent } from './vertical-nav.component';
 
-describe('VerticalNavComponent', () => {
+xdescribe('VerticalNavComponent', () => {
```
Comment (lbroudoux) [inline webapp/src/main/webapp/src/app/components/vertical-nav/vertical-nav.component.spec.ts:20]: Could you please remove this from the commit as it is not related to the purpose of the change?

File: webapp/src/main/webapp/src/app/pages/importers/importers.page.spec.ts [hunk 3]
```diff
@@ -17,15 +17,14 @@ import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 
 import { ImportersPageComponent } from './importers.page';
 
-describe('ServicesPageComponent', () => {
+xdescribe('ServicesPageComponent', () => {
```
Comment (lbroudoux) [inline webapp/src/main/webapp/src/app/pages/importers/importers.page.spec.ts:20]: Could you please remove this from the commit as it is not related to the purpose of the change?

File: webapp/src/main/webapp/src/app/pages/services/services.page.spec.ts [hunk 4]
```diff
@@ -17,15 +17,14 @@ import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 
 import { ServicesPageComponent } from './services.page';
 
-describe('ServicesPageComponent', () => {
+xdescribe('ServicesPageComponent', () => {
```
Comment (lbroudoux) [inline webapp/src/main/webapp/src/app/pages/services/services.page.spec.ts:20]: Could you please remove this from the commit as it is not related to the purpose of the change?

File: webapp/src/main/webapp/angular.json [hunk 5]
```diff
@@ -118,8 +118,7 @@
           "options": {
             "tsConfig": [
               "tsconfig.app.json",
-              "tsconfig.spec.json",
-              "e2e/tsconfig.json"
+              "tsconfig.spec.json"
```
Comment (lbroudoux) [inline webapp/src/main/webapp/angular.json:121]: Could you explain why is it necessary?

File: webapp/src/main/webapp/src/app/pages/tests/create/test-create.page.ts [hunk 6]
```diff
@@ -13,28 +13,34 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { Component, OnInit} from "@angular/core";
-import { ActivatedRoute, Router, ParamMap } from "@angular/router";
+import { Component, OnInit } from "@angular/core";
```
Comment (lbroudoux) [inline webapp/src/main/webapp/src/app/pages/tests/create/test-create.page.ts:16]: These are a lot of formatting changes that make it more difficult to review for merging. Please keep changes related to PR purpose.

**Review by lbroudoux — COMMENTED** [review]
Can you please review my comments please?

File: webapp/src/main/webapp/src/app/app.component.spec.ts (hunk 1 above)
Comment (soGit) [reply webapp/src/main/webapp/src/app/app.component.spec.ts:18]: Tests are not setup. Do you want me to create a separate MR for disabling/removing them?

File: webapp/src/main/webapp/angular.json (hunk 5 above)
Comment (soGit) [reply webapp/src/main/webapp/angular.json:121]: Project does not contain e2e/tsconfig.json file

**Comment by soGit** [comment]
> Can you please review my comments please?

fixed
